File: Python/Code/Aksesoris.py
from PetShop import PetShop
class Aksesoris (PetShop) :
    __jenis = ""
    __bahan = ""
    __warna = ""

    def __init__(self, ID, nama_produk, harga_produk, stok_produk, jenis, bahan, warna):
        super().__init__(ID, nama_produk, harga_produk, stok_produk)
        self.__jenis = jenis
        self.__bahan = bahan
        self.__warna = warna
    
    def getJenis(self) :
        return self.__jenis
    
    def getBahan(self) :
        return self.__bahan
    
    def getWarna(self) :
        return self.__warna

File: Java/Code/Aksesoris.java
public class Aksesoris extends PetShop{
    private String jenis;
    private String bahan;
    private String warna;

    public Aksesoris() {
    }

    public Aksesoris(String id, String nama_produk, int harga_produk, int stok_produk, String jenis, String bahan, String warna) {
        super(id, nama_produk, harga_produk, stok_produk);
        this.jenis = jenis;
        this.bahan = bahan;
        this.warna = warna;
    }

    public String getJenis(){
        return this.jenis;
    }

    public String getBahan(){
        return this.bahan;
    }

    public String getWarna(){
        return this.warna;
    }
}
